pom.xml

```
@@ -13,7 +13,7 @@
   <packaging>hpi</packaging>
   <name>Git plugin</name>
   <description>Integrates Jenkins with Git SCM</description>
-  <url>https://github.com/jenkinsci/git-plugin/blob/master/README.adoc</url>
+  <url>https://github.com/${gitHubRepo}</url>
```

No, revert this, or change to just
```suggestion
  <url>https://github.com/jenkinsci/git-plugin</url>
```

(You do _not_ want incremental builds from forked PRs to set their URL to the fork. This is only for `/project/scm`.)